site/src/components/WorkspaceDeletion/ImpendingDeletionBanner.tsx

```diff
@@ -20,21 +27,35 @@ export const ImpendingDeletionBanner = ({
   // is merged up
   const allowWorkspaceActions = experiments.includes("workspace_actions")
 
+  if (
+    !workspace ||
+    !displayImpendingDeletion(
+      workspace,
+      allowAdvancedScheduling,
+      allowWorkspaceActions,
+    ) ||
+    !displayImpendingDeletionBanner
+  ) {
+    return null
+  }
```

Comment:
I found it hard to read/understand this "display or not" logic here. I think the names `displayImpendingDeletion` and `displayImpendingDeletionBanner` are very similar and I took some time to understand they are different but not sure how to improve this.

Reply:
Ah, let me improve that